mert/SerScorer.cpp

```diff
@@ -69,19 +69,19 @@ void SerScorer::prepareStats(size_t sid, const string& text, ScoreStats& entry)
       }
       if (errors == 0) {
         n_ser = 0;
-	break;
+        break;
       }
     }
   }
   ostringstream stats;
-  stats << n_ser;
+  stats << n_ser << " " << 1; // sentence error (0 or 1), number of sentences (1)
```

Comment:
why do we need number of sentences?

Reply:
For conversion from sentence-level (0/1) to doc-level SER. This is used in calculateScore